frontend/localConfig.json

```diff
@@ -488,6 +488,7 @@
         "cfg": {
           "mapOptions": {
             "openlayers": {
+              "maxResolution": 33590,
```

Comment:
@dsuren1 is this limiting the zoom? Is it related to the zoom to extent fix?

Reply:
This is the fix for zoom to extent. Setting maxResolution helps us to calculate the resolutions correctly in our case. Hence the zoom happens as expected.